src/main/java/com/bean00/response/Response.java

```diff
@@ -58,6 +58,13 @@ public String getHeader(String header) {
         return headers.get(header);
     }
 
+    public int getContentLength() {
```

Comment:
Is this used anywhere besides tests?

Reply:
Not currently. I know the tests should support the code, and code shouldn't just be written for the tests, but in this case I like having the helper method to clean up the tests. I'll keep an eye on the `getContentLength` method, and if it isn't used I'll take it out and adjust the tests accordingly.